mirest/stylist-api
# #161700940 Setup travis and Heroku

#### Description
- Setup TravisCI and heroku
#### How Has This Been Tested?
- checkout into the branch ch-setup-travis-and-heroku-161700940
- run `python manage.py test`
- check out the URL https://stylist-api.herokuapp.com/
- Check out the badges on the readme file

### Pivotal tracker story
- [#161700940](https://www.pivotaltracker.com/story/show/161700940)
